tests/jobs/test_waiting.py

```diff
@@ -55,62 +59,21 @@ def test_wait_while_executing(sleep_mock):
     captured_output = io.StringIO()
     sys.stdout = captured_output
 
-    workflow_execution = mock.Mock()
-    execution_status = mock.Mock()
-    statuses = mock.PropertyMock(side_effect = ["InProgress", "Finished", "Finished"])
-    in_progress = mock.PropertyMock(side_effect = [True, False, False])
-    type(execution_status).status = statuses
-    type(execution_status).in_progress = in_progress
-    workflow_execution.status.return_value = execution_status
+    workflow_execution = mock.Mock(spec=WorkflowExecution)
+    statuses = mock.PropertyMock(side_effect=[
+        WorkflowExecutionStatus("InProgress", None),
+        WorkflowExecutionStatus("Succeeded", None),
+        WorkflowExecutionStatus("Succeeded", None)
+    ])
+    workflow_execution.status = statuses
 
     wait_while_executing(workflow_execution, print_status_info=True)
 
-    assert("Finished" in captured_output.getvalue())
-
-
-@mock.patch('time.sleep', return_value=None)
-def test_wait_while_executing_predictor_evaluation_execution(sleep_mock):
```

Comment:
Why the removal?

Reply:
Because the logic of `wait_while_executing` and `wait_while_validating` was collapsed into a single function, this now tests the exact same code as `test_wait_while_validating`